internal/platform/database/billing/repository.go

```
@@ -418,6 +632,254 @@ INSERT INTO accounting_entries (
+	var ok bool
+	query, args, ok = appendPropertyScope(query, args, scope, "p")
+	if !ok {
+		return nil, ErrNotFound
+	}
+	query += "ORDER BY mse.created_at ASC, mse.id ASC\n"
+
+	rows, err := r.db.QueryContext(ctx, query, args...)
+	if err != nil {
+		return nil, fmt.Errorf("get finalized financial report: %w", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `rows.Close` is not checked

(errcheck)